### DISCUSSION TITLE
Exceptions with helm charts for ARMO

### DISCUSSION THREAD

**Discussion by luk-kop** [kubescape/kubescape#525] [General]
Hi, is it possible to use exceptions with helm charts for ARMO cluster components in similar way as with kubescape cli tool? Instead of clicking it in UI, can I add/attach prepared json file with exceptions (or sth else) with helm charts?

This would be useful in case I have several clusters where I want to add the same exceptions. At the moment I have to click through manually in the UI. Thanks for reply.

**Comment by dwertent** [comment]
Hi and thank you for opening this discussion.
Currently, we do not support adding the exceptions file when running the helm chart. This should not be complicated to add, you are more than welcome to contribute it :)

Regarding your use case, you can submit your exceptions using the `kubescape submit exceptions`, make sure to include the `--account`, `--client-id`,`--secret-key` flags.
If you do not include the cluster name in the exceptions, the exceptions will apply to all clusters.

Please feel free to share any issues you have regarding this solution :)
